mule-extensions-api/src/main/java/org/mule/runtime/extension/api/util/ExtensionMetadataTypeUtils.java

```diff
@@ -98,6 +98,14 @@ public static String getId(MetadataType metadataType) {
   }
 
   /**
+   * @param type a {@link MetadataType}
+   * @return whether the given {@code type} represents an {@link InputStream} or not
+   */
+  public static boolean isInputStream(MetadataType type) {
+    return InputStream.class.isAssignableFrom(getType(type));
```

Comment:
`JavaType::getType(type)` throws exceptions if `type` was not loaded from an actual Java class. The Util method should not fail that badly and just return `false`, similar to what we do in the `getId`.